src/UtilFunctionTesters/RLCUtil/IsNumberTester.cpp

```diff
@@ -0,0 +1,22 @@
+#include "UtilFunctionTesters/RLCUtil/IsNumberTester.h"
+
+IsNumberTester::IsNumberTester() {}
+
+bool IsNumberTester::runTest() {
+    std::map<std::string, bool> expectedResultMap;
+    expectedResultMap["1"] = true;
+    expectedResultMap["1 "] = false;
+    expectedResultMap[""] = false;
+    expectedResultMap["1234567890123456789012345678901234567890"] = true;
```

Comment:
We should test at least one negative number and one float (i.e., number with a decimal point in it), as well, to encode the behavior in those cases.